### ISSUE TITLE
JOSM issue #15809 - issue with translated strings for scripting plugin? 

### ISSUE THREAD

**Issue by Gubaer** [Gubaer/josm-scripting-plugin#67]
see https://josm.openstreetmap.de/ticket/15809

**Comment by Gubaer** [comment]
@floscher : do you have an idea where this could stem from?

**Comment by Gubaer** [comment]
Probably  this first entry should be removed from the `.po` file.
```
msgid ""
msgstr ""
"Project-Id-Version: josm-plugin/josm-plugin_scripting 30780\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-01-17 21:36+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: floscher <florian@schaeferban.de>, 2018\n"
"Language-Team: German (https://www.transifex.com/josm/teams/2544/de/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: de\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

**Comment by floscher** [comment]
I released the new version `v0.3.2`.
It will ignore the entry for the empty string when generating *.lang files from *.po files.

Also you can now use `./gradlew transifexDownload` to update all existing languages in `src/main/po`. This will internally call the Transifex client and then cleans up the *.po files (e.g. remove the locations of each string in the source code).

If you want to call the transifex client manually, you should then run `./gradlew shortenPoFiles` once to also run the above mentioned cleaning steps.

**Comment by Gubaer** [comment]
Great, thanks!
